### PR TITLE
Update checkbox

### PR DESCRIPTION
**Pull request by swapnilraj**
image

image

Update checkbox in general settings under Discovery to stylised checkbox.

Signed-off-by: Swapnil Raj sr@sraj.me

### CODE REVIEW

File: src/components/views/terms/InlineTermsAgreement.js [hunk 1]
```diff
@@ -90,8 +91,9 @@ export default class InlineTermsAgreement extends React.Component {
                 <div key={i} className='mx_InlineTermsAgreement_cbContainer'>
                     <div>{introText}</div>
                     <div className='mx_InlineTermsAgreement_checkbox'>
-                        <input type='checkbox' onChange={() => this._togglePolicy(i)} checked={policy.checked} />
+                      <StyledCheckbox onChange={() => this._togglePolicy(i)} checked={policy.checked}>
```
Comment (t3chguy) [inline src/components/views/terms/InlineTermsAgreement.js:94]: indent in the area looks wrong

Comment (swapnilraj) [reply src/components/views/terms/InlineTermsAgreement.js]: Oops, should be fixed now!

**Comment by t3chguy** [comment]
It looks like you added the country code width commits into this branch too

**Comment by swapnilraj** [comment]
Should I force push to remove those commits?

**Comment by t3chguy** [comment]
That would be great!

**Pull request merged**